README.md

```diff
@@ -108,16 +108,16 @@ web3.api::<CustomNamespace>().custom_method().wait().unwrap()
 
 Currently, Windows does not support IPC, which is enabled in the library by default.
 To compile, you need to disable the IPC feature:
-```
-web3 = { version = "0.17.0", default-features = false, features = ["http"] }
+```toml
+web3 = { version = "_", default-features = false, features = ["http"] }
 ```
 
 # Avoiding OpenSSL dependency
 
 On Linux, `native-tls` is implemented using OpenSSL. To avoid that dependency
 for HTTPS use the corresponding feature.
-```
-web3 = { version = "0.17.0", default-features = false, features = ["http-rustls-tls"] }
+```toml
+web3 = { version = "_", default-features = false, features = ["http-rustls-tls", "signing", "ws-rustls-tokio", "ipc-tokio"] }
```

Comment:
Why add `signing` and `ipc-tokio` here? The purpose of this example is to explain how to avoid OpenSSL.
```suggestion
web3 = { version = "_", default-features = false, features = ["http-rustls-tls", "ws-rustls-tokio"] }
```

Reply:
This replicates the default features replacing openssl with rustls.

Reply:
I've changed it to mentioning the default features in an additional note below.